css/main.css

```diff
@@ -325,3 +325,60 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact-form {
+  padding: 10%;
+}
+
+#contact h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  border: none;
+  outline: none;
+  border-bottom: solid 2px #dbd8d7;
+}
+
+textarea {
+  border: none;
+  color: #3c3a39;
+  outline: none;
```

Comment:
[OPTIONAL]

- [ ] What do you think about applying the same background color as the one in the template to your project?
I'm sure the field will become more visible and also help users find the textarea field easily. 😁 

Here is the background color as seen in the template: ```#FBF8F7```